src/com/company/entities/Courier.java

```diff
@@ -0,0 +1,73 @@
+package com.company.entities;
+
+public class Courier {
+    private String name;
+    private String transport;
+    private Double raiting = 0.0;
```

Comment:
small nitpick, small typo :) ratings instead of raitings - do a replace all in the project of raitings with ratings

Reply:
fixed :)